### ISSUE TITLE
Validation Set

### ISSUE THREAD

**Issue by balakiki** [MAXNORM8650/MSFM#1]
Hi, sorry to bother you! I would like to know if you use a validation set when training with the MURA dataset? It seemed that you only mentioned the train and test set in the paper.

**Comment by MAXNORM8650** [comment]
I am so sorry for the delayed reply.

I only used MURA from the orgininal website.
